### PR TITLE
Added better and more descriptive logging

### PR DESCRIPTION
**Pull request by AnonymousBit0111**
- turned dbg!("todo") into todo!()
- turned dbg! statements into log::debug! statements
- added logging to pumpkin-inventory
- added logging when a client tries to interact with a block out of reach

### CODE REVIEW

File: pumpkin-inventory/Cargo.toml [hunk 1]
```diff
@@ -7,6 +7,9 @@ edition.workspace = true
 # For items
 pumpkin-world = { path = "../pumpkin-world" }
 
+simple_logger = { version = "5.0.0", features = ["threads"] }
```
Comment (Snowiiii) [inline pumpkin-inventory/Cargo.toml:10]: Thats definitely not needed here

**Comment by Snowiiii** [comment]
Also please remove the `todo!()` statements, i use them for not crashing the server

File: pumpkin-inventory/Cargo.toml (hunk 1 above)
Comment (AnonymousBit0111) [reply pumpkin-inventory/Cargo.toml:10]: you have a dbg! statement there so I changed it to log::dbg!
unless you're referring to the threads feature, that's probably not needed

**Comment by Snowiiii** [comment]
@AnonymousBit0111. Just want to tell you that you can use `cargo clippy --release` for checking for clippy warns and `cargo fmt` to format the code

**Comment by lukas0008** [comment]
nah these are my mistakes, im trying to fix merge issues on web interface

**Pull request merged**